### PR TITLE
Add documentation about GitHub Code Scanning.

### LINKED ISSUE #36: Add GitHub SARIF reporting workflow
cf https://3.basecamp.com/4878741/buckets/19340499/messages/3221129200#__recording_3222604782

### PR DESCRIPTION
**Pull request by mperrien**
Closes #36.

Signed-off-by: Marc Perrien <marc@chaosiq.io>

### CODE REVIEW

File: sources/start/github/index.md [hunk 1]
```diff
@@ -52,6 +52,38 @@ to the `Actions` tab on your repository on GitHub and you should see that your n
 
 <img src="../../static/images/gh-reliably-workflow-added.png" alt="New Reliably GitHub workflow added"/>
 
+## Enable Code Scanning Alerts
+Code Scanning is a GitHub feature which groups potential vulnerabilities in the Security tab of your repository. To enable Code Scanning Alerts, you need to:
+
+* tell the action to continue on error
+* output the errors to a SARIF report file
+* upload this SAIR report to GitHub Code Scanning
```
Comment (russmiles) [inline sources/start/github/index.md:60]: SAIR? If this is right can you include a link to what this means?

File: sources/start/github/index.md [hunk 2]
```diff
@@ -52,6 +52,38 @@ to the `Actions` tab on your repository on GitHub and you should see that your n
 
 <img src="../../static/images/gh-reliably-workflow-added.png" alt="New Reliably GitHub workflow added"/>
 
+## Enable Code Scanning Alerts
+Code Scanning is a GitHub feature which groups potential vulnerabilities in the Security tab of your repository. To enable Code Scanning Alerts, you need to:
```
Comment (russmiles) [inline sources/start/github/index.md:56]: Can you link to a definition of Code Scanning on GitHub as a feature description?

File: sources/start/github/index.md [hunk 3]
```diff
@@ -52,6 +52,38 @@ to the `Actions` tab on your repository on GitHub and you should see that your n
...
+
+* tell the action to continue on error
+* output the errors to a SARIF report file
+* upload this SAIR report to GitHub Code Scanning
+
+Your YAML workflow file should then look like this:
```
Comment (russmiles) [inline sources/start/github/index.md:62]: "An example YAML workflow file that applies this would include:"

File: sources/start/github/index.md [hunk 4]
```diff
@@ -111,6 +143,22 @@ Reliably has surfaced three Suggestions:
 
 All of these Reliably Suggestions *could* lead to reliability problems if you were to apply this manifest to your system.
 
+If you enabled Code Scanning Alerts and Reliably surfaces Suggestions, the Security tab in your repository will now display the number of suggestions.
```
Comment (russmiles) [inline sources/start/github/index.md:146]: "Alerts, and Reliably surfaces some Reliably Suggestions, then the Security..."

File: sources/start/github/index.md [hunk 5]
```diff
@@ -111,6 +143,22 @@ Reliably has surfaced three Suggestions:
...
+
+<img src="../../static/images/github-security-tab.png" alt="GitHub Security tab displays shows there are two Suggestions"/>
+
+Clicking on this tab will display the Security page for your repository. The lefthand menu will allow you to display the Code scanning alerts.
```
Comment (russmiles) [inline sources/start/github/index.md:150]: "Clicking on the `Security` tab"

File: sources/start/github/index.md [hunk 6]
```diff
@@ -111,6 +143,22 @@ Reliably has surfaced three Suggestions:
...
+
+<img src="../../static/images/security-page.png" alt="The lefthand menu allows you to display your code scanning alerts"/>
+
+All your Code scanning alerts will be listed there. Here, we have two alerts from Reliably.
```
Comment (russmiles) [inline sources/start/github/index.md:154]: "Code Scanning"

Comment (russmiles) [inline sources/start/github/index.md:154]: "two alerts prompted by Reliably Suggestions."

File: sources/start/github/index.md [hunk 7]
```diff
@@ -111,6 +143,22 @@ Reliably has surfaced three Suggestions:
...
+
+<img src="../../static/images/code-scanning-alerts-list.png" alt="A list of open code scanning alerts"/>
+
+Each alert is displayed in the context of the file which triggered it. In this example, our `manifest.yaml` file. The Suggestion from Reliably is also displayed.
```
Comment (russmiles) [inline sources/start/github/index.md:158]: "in which it was triggered."

Comment (russmiles) [inline sources/start/github/index.md:158]: "In this example, the `manifest.yaml` file."

Comment (russmiles) [inline sources/start/github/index.md:158]: "The Reliably Suggestion is also displayed."

**Review by russmiles — CHANGES_REQUESTED** [review]
As there are a reasonable amount of references to "Reliably Suggestion", this might deserve a link to its definition in our docs early on in the document if it's not already there?

File: sources/start/github/index.md (hunk 1 above)
Comment (mperrien) [reply sources/start/github/index.md]: Typo, of course.

**Comment by mperrien** [comment]
@russmiles For weird reasons, it still says there are changes requested, but I've fixed all the comments from the previous PR.

**Review by russmiles — APPROVED** [review]
LGTM!

**Comment by russmiles** [comment]
It was probably because I hadn't re-reviewed yet @mperrien :) Done now!

**Pull request merged**